react/features/prejoin/reducer.js

```diff
@@ -29,11 +29,23 @@ const DEFAULT_STATE = {
     userSelectedSkipPrejoin: false
 };
 
+const STORE_NAME = 'features/prejoin';
+
+/**
+ * Sets up the persistence of the feature {@code prejoin}.
+ */
+const filterSubtree = {
```

Comment:
These are no longer stored in the prejoin feature. Are you sure you are running latest master as the base for your PR?

Reply:
Missed that change when rebased, updating